rskj-core/src/test/java/org/ethereum/rpc/exception/handlers/UnsupportedOperationExceptionHandlerTest.java

```diff
@@ -0,0 +1,63 @@
+package org.ethereum.rpc.exception.handlers;
+
+import co.rsk.jsonrpc.JsonRpcError;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.googlecode.jsonrpc4j.ErrorResolver;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.slf4j.Logger;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.verify;
+
+class UnsupportedOperationExceptionHandlerTest extends ExceptionHandlerTestBase {
+
+    @Mock
+    private Logger logger;
+
+    private UnsupportedOperationExceptionHandler errorHandler;
+    private AutoCloseable autoCloseableMocks;
+
+    @BeforeEach
+    void setUp() {
+        autoCloseableMocks = MockitoAnnotations.openMocks(this);
+        errorHandler = new UnsupportedOperationExceptionHandler() {
+            @Override
+            protected void logJsonRPCError(Throwable throwable, Method method, List<JsonNode> arguments) {
+                logger.error(getLogMessage(method, arguments), throwable);
+            }
+        };
+    }
+
+    @AfterEach
+    void tearDown() throws Exception {
+        autoCloseableMocks.close();
+    }
```

Comment:
Why do you need to add this in all mocked class? Aren't they independent?

If you need always to do that anyway, why not put these classes on the `ExceptionHandlerTestBase` and let it do it for you for all inherited classes?